README.md

```diff
@@ -156,6 +156,26 @@ Once an successful installation of Che/CRW is verified, tests patch custom resou
 * enable oAuth
 * enable TLS mode
 
+You can check and turn ono/ff the TLS mod from ConfigMap by following the steps:
+1. See ConfigMap
+```sh
+oc get configmap
+```
+
+![oc get configmap](./images/TLSEnabledFlag_1.jpg)
+
+2. Describe or edit Configmap che to see the value **CHE_INFRA_OPENSHIFT_TLS_ENABLED**
+```sh
+oc edit configmap che
+```
+or
+```sh
+oc describe configmap che
```

Comment:
what about:

```
oc describe configmap che| grep CHE_INFRA_OPENSHIFT_TLS_ENABLED
```

So you only see what you want to see? Then step 3 is no longer needed.